api/src/main/java/org/pmiops/workbench/reporting/ReportingJobResult.java

```diff
@@ -2,7 +2,6 @@
 
 public enum ReportingJobResult {
   SUCCEEDED,
-  ABORTED,
```

Comment:
?

Reply:
This was just a placeholder. After the alpha delivery, I want to implement an actual error [handling/reporting/mitigating strategy](https://precisionmedicineinitiative.atlassian.net/browse/RW-5384).